packages/test/test-end-to-end-tests/src/test/localLoader.spec.ts

```diff
@@ -287,9 +288,11 @@ describeNoCompat("LocalLoader", (getTestObjectProvider) => {
 
                 container1 = await createContainer(documentId, testDataObjectFactory);
                 dataObject1 = await requestFluidObject<TestDataObject>(container1, "default");
+                await waitContainerToCatchUp(container1);
```

Comment:
I have seen issues like these too where the loader container tracker hands if the container is not connected.
nit: You could use `waitForContainerConnection` in test-utils if you want to wait for connected only.

Reply:
https://dev.azure.com/fluidframework/internal/_boards/board/t/Team%20Curtis/Features/?workitem=2005